cohorts/views.py

```diff
@@ -134,6 +137,354 @@ def get_cases_by_cohort(cohort_id):
         if cursor: cursor.close()
         if db and db.open: db.close()
 
+
```

Comment:
Since the methods below aren't views, it'd be better if they were moved into a module of their own--API helpers, for example.